### ISSUE TITLE
Latex Rendering issue

### ISSUE THREAD

**Issue by shingtaklam1324** [leanprover-community/doc-gen#62]
https://leanprover-community.github.io/mathlib_docs/data/mv_polynomial/basic.html

![image](https://user-images.githubusercontent.com/15098580/92995346-67b1d180-f535-11ea-8083-2b0c4e1abc47.png)

The source is

```
might denote $R[X_i : i \in \sigma]$. It is the type of multivariate
```

which looks like valid Latex to me.

**Comment by digama0** [comment]
The source on the website says
```
might denote $R[X_i : i \in sigma]$. It is the type of multivariate
```
so this is not mathjax's fault.

**Comment by bryangingechen** [comment]
We should investigate and fix the underlying issue, but for this particular example I'm surprised we didn't just use the unicode `σ`.

**Comment by alexpeattie** [comment]
Looks like a weird `markdown2` bug. Minimal reproduction:

```python
>>> import markdown2

>>> markdown2.markdown('Example: $x \in \sigma$ and separately `y`')
'<p>Example: $x \\in \\sigma$ and separately <code>y</code></p>\n'

>>> markdown2.markdown('Example: $x \in \sigma$ and separately `s`')
'<p>Example: $x \\in sigma$ and separately <code>s</code></p>\n'

>>> markdown2.markdown('Example: $x \in \sigma$ and separately `sigm`')
'<p>Example: $x \\in sigma$ and separately <code>sigm</code></p>\n'

>>> markdown2.markdown('Example: $x \in \sigma$ and separately `simg`')
'<p>Example: $x \\in \\sigma$ and separately <code>simg</code></p>\n'

>>> markdown2.markdown('Example: separately `s` and $x \in \sigma$')
'<p>Example: separately <code>s</code> and $x \\in sigma$</p>\n'
```

Escaping in a backslash escaping seems to break if there's a `<code>` tag before or afterwards containing a prefix of the escaped word 🙈 . I can't see any reason why that would be. In this example it's because of the ``s`` [here](https://github.com/leanprover-community/mathlib/blob/d5be9f388a19f891daed609add98153b3024c100/src/data/mv_polynomial/basic.lean#L51), remove that and the escaping on `\sigma` works as expected.

**Comment by alexpeattie** [comment]
I've tracked down a few other places this is happening:

* [`analysis/analytic/basic.html`](https://leanprover-community.github.io/mathlib_docs/analysis/analytic/basic.html#mod_doc) (doesn't render anyway due to an unrelated issue with `\choose`)
* [`analysis/calculus/inverse.html`](https://leanprover-community.github.io/mathlib_docs/analysis/calculus/inverse.html#approximates_linear_on.inverse_continuous_on)
* [`dynamics/circle/rotation_number/translation_number.html`](https://leanprover-community.github.io/mathlib_docs/dynamics/circle/rotation_number/translation_number.html#circle_deg1_lift.tendsto_translation_number)
* [`representation_theory/maschke.html`](https://leanprover-community.github.io/mathlib_docs/representation_theory/maschke.html#equivariant_projection)

**Comment by gebner** [comment]
This should be fixed now by #110
